test/functional/apps/visualize/_experimental_vis.js

```diff
@@ -50,23 +50,6 @@ export default ({ getService, getPageObjects }) => {
         expect(await info.getVisibleText()).to.contain('experimental');
       });
 
-      it('should show an notification when creating lab visualizations', async () => {
-        // Try to find a lab visualization.
-        const labTypes = await PageObjects.visualize.getLabTypeLinks();
```

Comment:
should probably also remove `getLabTypeLinks` here:
https://github.com/elastic/kibana/blob/05086f320dcb7539101d53243cbe9bf63a20f41b/test/functional/page_objects/visualize_page.js#L197-L199